.github/workflows/build.yml

```diff
@@ -8,30 +8,38 @@ on:
     
 jobs:
   tests:
-    runs-on: macos-12
+    runs-on: ${{ matrix.os }}
```

Comment:
Nice :)

Reply:
is this for parallel exceution?

Reply:
[Correct](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs) :)